android/src/main/java/com/reactlibrary/JudoReactNativeModule.java

```diff
@@ -67,16 +67,52 @@
     private static final int PAYMENT_REQUEST = 101;
     private static final int PRE_AUTH_REQUEST = 201;
     private static final int LOAD_PAYMENT_DATA_REQUEST_CODE = 1337;
+    private static final int GPAY_REQUEST = 500;
     private static final String ACTIVITY_DOES_NOT_EXIST = "ACTIVITY_DOES_NOT_EXIST";
     private static final String JUDO_ERROR = "JUDO_ERROR";
     private static final String JUDO_GOOGLE_PAY_ERROR = "JUDO_GOOGLE_PAY_ERROR";
     private static final String JUDO_USER_CANCELLED = "JUDO_USER_CANCELLED";
+    private static final String SDK_WRAPPER_NAME = "RNJudo";
+    private static final String PAYMENT_METHODS_KEY = "paymentMethods";
 
     private final String ISO_FORMAT = "yyyy-MM-dd'T'HH:mm:ss'Z'";
     private final SimpleDateFormat sdf = new SimpleDateFormat(ISO_FORMAT, Locale.ENGLISH);
     private Promise promise;
     private ReadableMap options;
     private Disposable disposable;
+    private PaymentsClient googlePayClient;
+
+    private static final String PAYMENT_METHOD_NONE = "PAYMENT_METHOD_NONE";
+    private static final String PAYMENT_METHOD_CARD = "PAYMENT_METHOD_CARD";
+    private static final String PAYMENT_METHOD_GOOGLE_PAY = "PAYMENT_METHOD_GOOGLE_PAY";
+    private static final String PAYMENT_METHOD_ALL = "PAYMENT_METHOD_ALL";
+
+    @Nonnull
+    @Override
+    public String getName() {
+        return JudoReactNativeModule.SDK_WRAPPER_NAME;
+    }
+
+    @Override
+    public void onHostResume() {
+        // Activity `onResume`
+    }
+
+    @Override
+    public void onHostPause() {
+        // Activity `onPause`
+    }
+
+    @Override
+    public Map<String, Object> getConstants() {
+        final Map<String, Object> constants = new HashMap<>();
+        constants.put(PAYMENT_METHOD_NONE, 0);
+        constants.put(PAYMENT_METHOD_CARD, 1);
+        constants.put(PAYMENT_METHOD_GOOGLE_PAY, 2);
+        constants.put(PAYMENT_METHOD_ALL, 3);
```

Comment:
I expose all the flags that are going from Judo SDK

Reply:
These should return values as strings, eg:
`constants.put(PAYMENT_METHOD_NONE, PAYMENT_METHOD_NONE)` this to avoid the issue of adding new payment methods (instead of number `4` for a new method which would be weird after "ALL" as 3, it would have a proper string value). I'm not sure I can be any clearer than this.

Reply:
If there will be a need of adding one more value `ALL` will be moved and in place of 3 will be another value. Client won't be affected.

Reply:
No, the client won't be affected as long as they use our exported constants. My point is - what if they don't. Also, changing "const" values is something I'd like to avoid in future releases.

Reply:
👍